simulation/simulation_interface/proto/simulation_api_schema.proto

```diff
@@ -33,6 +33,18 @@ message PseudoTrafficLightDetectorConfiguration {
   string architecture_type = 1;        // Autoware architecture type.
 }
 
+/**
+ * Parameter configuration of the imu sensor
+ **/
+ message ImuSensorConfiguration {
+  string entity = 1;                   // Name of the entity which you want to attach imu.
+  bool add_gravity = 2;                // If true, gravity will be added to the acceleration vector
+  bool add_noise = 3;                  // If true, noise will be added to every vector published
```

Comment:
This may be redundant, since turning noise off can be achieved by setting the noise parameter to zero.